pkg/graph/schema.graphql

```diff
@@ -125,6 +125,36 @@ type CedarDataCenter {
   zip: String
 }
 
+"""
+The possible types of assignees for CedarRoles
+"""
+enum CedarAssigneeType {
+  PERSON
+  ORGANIZATION
```

Comment:
While I can see the `enum` in the swagger file says this is the only two, we also have code to potentially return `undefined`. Should we add that here, or just remove the `undefined` check in `client.go`?

Reply:
After discussion, decided to ignore responses from CEDAR that have an invalid value for `AssigneeType`, instead of having an undefined value in the Go enum.

https://github.com/CMSgov/easi-app/pull/1438/commits/f88e5cb415cb4a706a1877d504fe080b278c8ce9 has the change to decoding.